lib/ram_pwrdn/ram_pwrdn.c

```diff
@@ -4,26 +4,33 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-#include <hal/nrf_power.h>
 #include <logging/log.h>
 #include <sys/util.h>
 #include <zephyr.h>
 #include <stdint.h>
 
+#if defined(CONFIG_SOC_NRF52840) || defined(CONFIG_SOC_NRF52833)
+#include <hal/nrf_power.h>
+#elif defined(CONFIG_SOC_NRF5340_CPUAPP)
+#include <hal/nrf_vmc.h>
+#else
+#error "RAM power-down module is not supported on the current platform"
+#endif
+
 #if CONFIG_RAM_POWER_ADJUST_ON_HEAP_RESIZE
 #include <init.h>
 #include <sys/heap_listener.h>
 #endif
 
-LOG_MODULE_REGISTER(ram_pwrdn, CONFIG_RAM_POWERDOWN_LOG_LEVEL);
-
-#if !defined(CONFIG_SOC_NRF52840) && !defined(CONFIG_SOC_NRF52833)
-#error "RAM power-down module is not supported on the current platform"
```

Comment:
RAM power-down module sounds perfect :)
However it seems like we call this library RAM Power Down in the rst file... RAM power-down would be better. Either way, we should be consistent.

Reply:
Changed!